src/apps/content-editor/src/app/views/ItemList/index.tsx

```diff
@@ -182,24 +193,16 @@ export const ItemList = () => {
 
   const computeProcessedItems = useCallback(() => {
     if (!items || isFieldsFetching || isUsersFetching) return [];
-    const fieldMap = fields?.reduce((acc, field) => {
-      // @ts-ignore
-      acc[field.name] = field.datatype;
-      return acc;
-    }, {});
+
+    const userMap = new Map(users?.map((user) => [user.ZUID, user]));
```

Comment:
Is there a significant performance gain to using Map instead of an object/array?